<!doctype html>
<html lang="en">

	<head>
		<meta charset="utf-8">

		<title>Network Motifs</title>

		<meta name="description" content="Understanding Biology Through Mathematics">
		<meta name="author" content="Laura Martens & Arthur Heimbrecht">

		<meta name="apple-mobile-web-app-capable" content="yes">
		<meta name="apple-mobile-web-app-status-bar-style" content="black-translucent">

		<meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=no">

		<link rel="stylesheet" href="css/reveal.css">
		<link rel="stylesheet" href="css/theme/moon.css" id="theme">

		<!-- Theme used for syntax highlighting of code -->
		<link rel="stylesheet" href="lib/css/zenburn.css">

		<!-- Printing and PDF exports -->
		<script>
			var link = document.createElement( 'link' );
			link.rel = 'stylesheet';
			link.type = 'text/css';
			link.href = window.location.search.match( /print-pdf/gi ) ? 'css/print/pdf.css' : 'css/print/paper.css';
			document.getElementsByTagName( 'head' )[0].appendChild( link );
		</script>
        <style type="text/css">
            .inner {
              position: absolute; right: 100px; bottom: -30px
            }
            .inner_2 {
              position: absolute; margin: 0 auto;
            }

        </style>

		<!--[if lt IE 9]>
		<script src="lib/js/html5shiv.js"></script>
		<![endif]-->
	</head>

	<body>

		<div class="reveal">

			<!-- Any section element inside of this container is displayed as a slide -->
			<div class="slides">
				<section data-background="img/network3.png" data-state="img-transparent" id="title">
					<div class="bodyneeded">
						<h1>Network Motifs</h1>
						<h3>Understanding Biology through Mathematics</h3>
						<p>
							<small class="subtitle">presented by Laura Martens & Arthur Heimbrecht</small>
						</p>
					</div>
				</section>

                
<!--Contents-->     
                
				<section >
					<h2>Contents</h2>
                    <p>Introduction</p>
                    <p>Biology</p>
                    <p>Large Networks</p>
                    <p>Conclusion & Outlook</p>
                    
					<aside class="notes">
						Oh hey, these are some notes. They'll be hidden in your presentation, but you can see them if you open the speaker notes window (hit 's' on your keyboard).
					</aside>
				</section>

<!--Introduction-->                
                
				<section id="introduction">
                    
                    
                    <!-- Folie 1 -->
					<section data-background="img/Laura/cell.jpg" data-state="img-transparent">
						<h2>Introduction</h2>
					</section>
                    
                    <!-- Folie 2 -->
					<section data-transition="slide">
						<div class="row">
							<h2>What are network motifs?</h2>
                            
								 <ul> 
                                    <li> <p class="fragment fade-up"> basic interaction patterns that recur throughout biological networks <br> much more often than in random networks </p> </li> 
                                 </ul> 
                            
                                <p class="fragment current-visible"> <img width="715" height="400" data-src="img/Laura/motif2.png" style="background:none; border:none; box-shadow:none;" > </p>
                            
						</div>
					</section>
                    
                    <!-- Folie 2 continued-->
                    
                    <section data-transition="slide">
						<div class="row">
                            
                            <div class="column" class = "fragment fade-up" >	 
                                <h2 > Real network</h2>
                                <br>
                                <img width="533" height="527" data-src="img/Laura/random_selfmade.png" alt="Down arrow">
                                
                                <img class = "fragment" width="200" height="184" data-src="img/Laura/Pfeil.png" style="background:none; border:none; box-shadow:none; position: absolute; left:860px; top:350px" data-fragment-index= "2">
                            </div>
                            
                            <div class="column" class = "fragment" data-fragment-index= "1">	 
                                <h2> Random network</h2>
                                <br>
                                <img width="533" height="527" data-src="img/Laura/real_selfmade.png" alt="Down arrow">      
                            </div>
                            
                            
						</div>
					</section>
                    
                    <!-- Folie 3 -->
					<section>
                       
						<div >
							<h2>Network motifs as building blocks</h2> 
                    
                            
                       
                           <ul>
                                
                               <li class = "fragment" data-fragment-index= "1"> <p> Each network has a defined function: e.g. filters, pulse generators </p>  </li> 
                            <li class = "fragment" data-fragment-index= "2"> <p>Small set of network motifs serve as bulding blocks of large networks</p> </li>
                            </ul> 

                            
                       
                            
                        </div> 
                            
					</section>
                    
                    <section>
                    
                         <div style = "position:absolute; top:20px;">
                            <img class = "fragment grow" data-fragment-index = "1" width="885" height="500" data-src="img/Laura/transcription_network.png" style="background:none; border:none; box-shadow:none;" >
                            <img class = "fragment shrink" data-fragment-index = "1" width="885" height="500" data-src="img/Laura/neurons.png" style="background:none; border:none; box-shadow:none;" >
                            <img class = "fragment shrink" data-fragment-index = "1" width="885" height="500" data-src="img/Laura/www.png" style="background:none; border:none; box-shadow:none; " >
                        </div>
                    </section>

                    
                     <!-- Folie 4 -->
                    
                    <section>
						
							<h2>Transcription networks</h2> 
                            <img width="952" height="750" data-src="img/Laura/trans_trans.png" style="background:none; border:none; box-shadow:none;" >

						
					</section>
                    
                    <section>
                        
                        <h2>Activators and repressors</h2>
                        <div style = "position: absolute; left: 450px; bottom:auto"><img class = "fragment current-visible" data-fragment-index = "0" width="996" height="525" data-src="img/Laura/activator.png" style="background:none; border:none; box-shadow:none;" ></div>
                        <div style = "position: absolute; left: 180px;"><img class = "fragment" data-fragment-index = "1" width="1572" height="400" data-src="img/Laura/repressor.png" style="background:none; border:none; box-shadow:none;" ></div>
                    
                    </section>
                    
                   
                    
                    <!-- Folie 5 -->
                    <section><h2>Evolution of network motifs</h2></section>
                    <section>

                            <div class = "inner_2"><img  width="1810" height="980" data-src="img/Laura/Evolution_1.png" style="background:none; border:none; box-shadow:none;" ></div>
                            <div class = "inner_2"><img class = "fragment"  width="1810" height="980" data-src="img/Laura/Evolution_2.png" style="background:none; border:none; box-shadow:none;" ></div>
                            <div class = "inner_2"><img class = "fragment"  width="1810" height="980" data-src="img/Laura/Evolution_3.png" style="background:none; border:none; box-shadow:none;" ></div>
                            <div class = "inner_2"><img class = "fragment"  width="1810" height="980" data-src="img/Laura/Evolution_4.png" style="background:none; border:none; box-shadow:none;" ></div>
                            <div class = "inner_2"><img class = "fragment"  width="1810" height="980" data-src="img/Laura/Evolution.png" style="background:none; border:none; box-shadow:none;" ></div>
                    

                    </section>
                    
                    

				</section>

<!--Mathematical models-->   
                
				<section id="biology">
					<section data-background="img/Laura/dna2.jpg" data-state="img-transparent" >
						<h2>Mathematical models</h2>
<!--						<p>Buzzers, Sniffers, Switches, Blinkers</p>-->
						<aside class="notes">
							Now we will get to the part, that all physicists were probably anticipating. We will talk about the mathematical models that describe network motifs quantitatively. But we will also talk about the qualitative aspects of certain network motifs and which functions they bring to a network.
							Ultimately our goal is to shed light onto the functioning of network motifs through a mathematical description.
						</aside>
					</section>
					<section>
						<h3>Preliminary example</h3>
						<iframe data-src="http://localhost:5006/diagram1a", width=1550, height=650px style="background: #FFFFFF;"></iframe>
						<aside class="notes">
							But first let's talk about a simple example.
							You see we are given a first order differential equation that describe the wire diagram on the top left.
							A first look at the wire diagram gives us an overview of what we are talking about:
							There are two Quantities - a Signal S and a Response R- that are correlated by those "wires" in between.
							Solid arrows represent either the aggregation of the quantity they are pointing to or the depletion of the quantity which they are pointing away from. Simply speaking they indicate where stuff is going.
							This would mean in a biological sense that protein R is synthesized through an arrow pointing to it and degraded by an arrow pointing away.
							Now there is also a second type of arrow, which is dashed. The dashed arrow means that one quantity influences a solid arrow in its magnitude.
							Again, speaking in a biological sense this means that one protein (the signal) influences the synthetization rate of the response protein.
							This information can be combined into a short differential equation that takes the aggregation (blue) and the depletion (red) according to the law of mass action into account.
							For your understanding: k0 is a base aggregation rate and the depletion always needs some R to be present in order to deplete it.

							Since the DE qualifies as an ODE we can solve it quite easily, which gives us the so called 'steady state' below.
							This depends only on the coefficients...and the Signal S.
							The plot to the right shows the calculated steady state solution and you can observe a linear dependency between the Signal and the steady state of the response.
							Keep in mind that these steady states are stable and as such the response will always tend towards the steady state for each signal, no matter what the current value of the response is.

							In this graph we can play around a little and change the values of some variables and as you can see, an increase in aggregation rate causes the response curve to be steeper while an increase of the depletion rate causes the opposite.
							In the left graph you can see how these effects are caused.
							The point in which both the aggregation rate and the depletion rate meet is the steady state since aggregation and depletion are equally large and thus dR/dt is zero. When changing the coefficients of each curve we observe the expected behavior.
							The steady state moves along both lines as the blue line changes due to the signal.

							An example for this would be S the concentration of mRNA and R being the protein concentration.

							This is the point I want to ask you if there are any questions concerning these principles.
							It is very important that you understand this very visualization in its entirety in order to be able to understand what is about to come.

							[answer questions]

							Now we will see, where to go from here.
							If you have any questions regarding those plots please do not hesitate to ask right away.
						</aside>
					</section>
					<section>
						<h3>Biological example</h3>
						<iframe data-src="http://localhost:5006/diagram1b", width=1550, height=650px style="background: #FFFFFF;"></iframe>
						<aside class="notes">
							Since this seminar also features biologists I want to present you another preliminary example that is actually taken from biology.
							And which process to choose but phosphorylization. The motor of our cells.
							What we observe in the wire diagram is the phosphorylization of our response protein such that is becomes RP.
							This uses up ATP and is controlled by a signal protein.
							Under the presence of water our phosphorylated protein looses the phosphor ion and goes back to its original state.
							If we put this into formulas we see that the rate of phosphorylation depends on the amount of unphorphorylated protein. We describe this with the difference of all protein that is present minus the phosphorylated amount.
							Please be careful as we are interested in the amount of phosphorylated protein RP not R. So all graphs that say R do in fact mean RP.
							On the left graph you can see the aggregation curve and the depletion curve respectively and the point where they meet, which is the steady state.

							Again we are given an ODE and can solve for R such that we get our steady state, which is plotted on the right hand side.

							If we compare this to the previous graph we see that the fact that the aggregation is not constant anymore, but depends on R as well gives us a hyperbolic shape.

							So, quick question to all biologists in the room, do you use this model for any calculations? Since obviously it is quite simplified.
							That is why we also have the more accurate model that is governed by Michaelis-Menten kinetics.
							It simply takes into account that rates can max out for processes no matter how strong the signal since [protein] << [ATP].
						</aside>
					</section>
					<section>
						<h3>Buzzer</h3>
						<iframe data-src="http://localhost:5006/diagram1c", width=1550, height=650px style="background: #FFFFFF;"></iframe>
						<aside class="notes">
							So now we are at the point where we can describe actual processes in biology.

							First of, we see that the differential equation is tweaked by a denominator that consists of the Michaelis factor which is constant and also the same variable as in the numerator.
							This causes the rates to take on a hyperbolic shape.
							We can still solve this quadratic ODE ... or rather look up the solution on the internet or in a paper and we are given the solution below the equation. The so called 'Goldbeter-Koshland' function.
							You don't need to remember this equation exactly although we will use is quite often. But you should remember that is has a sigmoidal shape similar to x/sqrt(1+x^2).
							But most important: sigmoidal shape!
							Again we can play around a bit and see what different coefficients cause on the plot.
							But let's look further into the shape of our steady state solution.
							As you may have noticed or already know, the sigmoid function if often used in machine learning for instance, since its shape is quite beneficial.
							For low signals we have a rather constant low response and for large signals we have a relatively constant large response. In between there is a steep ascent. 
							Still this curve is graded and reversible like its predecessors which means that it is monotonous and continuous (graded) and will give always the same value to an S no matter the previous value of S. 

							Because this function is in its properties relatively similar to a button -like on a doorbell- that there is no response when the button is not pressed but a constant response if the button is pressed.
							Because of this, this network motif is also called a 'Buzzer'. Another name is also zero-order ultrasensitivity.
							Its properties can be very important for gene expression or protein synthesis like.....TODO:add an example

						</aside>
					</section>
					<section>
						<h3>Sniffers</h3>
						<iframe data-src="http://localhost:5006/diagram1d", width=1550, height=650px style="background: #FFFFFF;"></iframe>
						<aside class="notes">
							The next motif is called a Sniffer.
							As the name suggests it is related to the our olfactory sense. Specifically that we are able to smell -be it good or bad- odors and no matter how strong they are at some point we get used to them.
							Same goes for this motif. Independent of the signal strength the response will always got back to a constant steady state. As you can see the plot clearly features a constant line as steady state, while both the depletion and the aggregation change with the signal they always cancel each other at the same response level.
							This is achieved by inserting a second quantity into the system - here X.
							X may be another protein or a ligand that connects to receptors.
							The signal increases both the aggregation of X and R but X also raises the depletion rate of R thus keeping R on the same level.
							If we were to look at this differential equation through time the graph would look something like this:
							For a step like increase of S, X slowly reaches its own steady state, which is in fact dependent on S!!
							At the same time R increases abruptly due to the increase in S but the slow adaption of X.
							It then comes back to the steady state as X reaches its final level.
							Repeating this process has the consequence that the peak in R gets smaller and smaller, since the height of the peak depends on the relation of the old aggregation rate and the new one.
							This is also called feed-forward control.
							[draw on blackboard]
							This motif is also called perfect adaption for which the reasons are obvious I think.
							This mechanism is not only used in our olfactory sense but also in chemotactic systems which adapt to a constant signaling level.
						</aside>
					</section>
					<section>
						<h3>Irreversible switch</h3>
						<iframe data-src="http://localhost:5006/diagram1e", width=1550, height=650px style="background: #FFFFFF;"></iframe>
						<aside class="notes">
							Now we get to a network motif that is closely connected to this seminars title. Namely the part about "cell fate decisions".
							They differ to the others that now we introduce feedback, you will see that in a second.
							This motif is called "irreversible switch" and works through positive feedback.
							The general thought behind this is that -again- we have a second quantity besides R and S but this time R amplifies the aggregation of EP and EP in return amplifies the aggreagtion of R.
							This works up to a certain point, where it reaches a critical signal level that causes R and EP to amplify each other stronger than they are depleted ultimately reaching a new steady state that is independent of S.
							This is important since now the response stays high even though the signal may fall back to zero, the decision is irreversible.
							The best example for this is apoptosis which can not be stopped once the signal for a cells death is given.

							If we look into the derivative plot we see how this can be achieved:
							The aggregation consists of the steady state solution for phosphorylation which applies to E and EP and also the amplification through S. For this reason we see a sigmoidal shape that is lifted according to S
							The depletion on the other side depends only on R and thus is a linear plot.
							This causes either one or three crossings where the total derivative is 0 and therefore we have a steady state.
							Now only the outer steady states are so called stable ones as the vector field around them points towards those points. It can also be explained as the aggregation curve cutting the depletion from above meaning that if we were to have a slightly lower R the aggregation would be larger than the depletion thus driving R towards the steady state and vice versa for a slightly too large R. Only if the system were to inhibit the exact state where dR/dt = 0 it would be a steady state but an instable one.

						</aside>
					</section>
					<section>
						<h3>Reversible switch</h3>
						<iframe data-src="http://localhost:5006/diagram1f", width=1550, height=650px style="background: #FFFFFF;"></iframe>
						<aside class="notes">
							In contrast to that there is a reversible switch or toggle switch that utilizes a negative feedback loop. You can see in the wiring diagram that R amplifies the aggregation of EP which is also depletion of E and E amplifies the depletion of R.
							So we get quite an interesting picture in the derivative plot in a way that the depletion hat this early peak and then comes back to a linear function but is independent of S.
							At the same time the aggregation depends solely on S and is rather constant. Hence we get one or three crossing ergo steady states depending on S. For one crossing it is obvious that the response will tend to this single steady state.
							But if there are multiple crossings the closest steady state to the previous one is taken up until the point where the next stable steady state is the only remaining one and we see a jump in the response. This process is reversible and gives us a switch that has different threshold for activation and deactivation.
							This results in hysteresis when inspecting the R-S-plot.
							examples would be:
							Activation of the M-phase promoting factor in frog egg axtracts. Mitosis is started but can be stopped if necessary.
							The autocatalytic conversion of normal prion protein to its pathogenic form. (BSE?) unintuitive since prions do not fold back, usually...
							And the lactose operon in bacteria, which induces lactose metabolism if no glucose is present.

						</aside>
					</section>
					<section>
						<h3>Blinker</h3>
						<iframe data-src="http://localhost:5006/diagram2a", width=1550, height=650px style="background: #FFFFFF;"></iframe>
						<aside class="notes">
							There is another "classic" network motif which causes homeostasis meaning that R is more or less static independent of S. But we have to keep track of time.

							Now we get into a more advanced territory were there is no longer only one or two stable steady states but we will observe oscillations.
							So how is this achieved?
							First off we link two phosphorylation processes to one another and a quantity X, which brings us to three quantities.
							Now we assume a constant signal which amplifies aggregation of X. This in return amplifies aggregation of YP with some delay which itself causes amplification of RP aggregation, again with a delay.
							R then attenuates the aggregation of X which again cascades down to the other two. This happens over and over if S is not too large.
							Looking at the signal response graph we see that the amplitude of the oscillation first grows with S, then it is shifted upwards until the amplitude shrinks back to an ordinary steady state.
							You can see that sustained oscillations do in fact depend on the Signal strength and for small or large enough S there is only a single steady state.
							At critical signal strength we see so called Hopf bifurcations, where the steady state changes into steady oscillations.
							If we look at this process under the fluorescence microscope we can see that the aggregation an depletion of RP as blinking of its marker.
							Thus, this is called a Blinker.
							It plays an important role in the circadian rhythm and again the maturation of cells and mitogen-activated protein kinase.
						</aside>
					</section>
					<section>
						<h3>Positive Feedback oscillator</h3>
						<iframe data-src="http://localhost:5006/diagram2b", width=1550, height=650px style="background: #FFFFFF;"></iframe>
						<aside class="notes">
							The next motif is a different oscillator that again has two additional quantities to S and R but one gives a positive feedback the other a negative.
							It combines a toggle switch mechanism with homeostasis as a positive and a negative feedback loop a present.
							Basically the idea is that the homeostatis mechanism drives the system back and forth between the toogle states of the switch.
							We express this as equations in the usual manner and then look specifically at the X-R-plane not dR/dt!
							We see that aggregation and depletion only cross at an instable steady state and thus we see a circular motion around this point along the vector field.
							This causes an oscillation in X and R.
							As we are only interested in R we see on the right side that for S between its critical values R is oscillating.
							This is for example used in cyclic AMP production in slime molds. cAMP binds to a receptor that releases more cAMP that again binds to the other receptors. This pushes the receptors into an inactive form which they recover from slowly only after the cAMP is removed.
						</aside>
					</section>
					<section>
						<h3>Negative Feedback oscillator</h3>
						<iframe data-src="http://localhost:5006/diagram2c", width=1550, height=650px style="background: #FFFFFF;"></iframe>
						<aside class="notes">
							We can implement this in a different way as well. Here R is synthesized from X and EP gives a positive feedback loop.
							For large enough S, X becomes large enough that R reaches a critical point where the positive feedback loop kicks in and all of X is synthesized to R, which increases R further to this point.
							Then R naturally depletes since there is no more X left and this causes EP to deplete as well. Ultimately, only very little X is converted into R and E can aggregate until it causes R to reach the critical value again.
							Important to this mechanism is the sudden start of oscillations at critical signal strength. This is contrary to the small amplitude oscillations for Hopf bifurcations in the first example.
							We do distinguish between those different kinds of Hopf bifurcations and call them supercritical in the first case and subcritical in the current case.
							subcritical systems have the advantage of generating an immediate and robust response at critical signaling strengths.

							
							This kind of oscillation can be observed in MPF where also the depletion of the substrate causes oscilaltions.
						</aside>
					</section>
					<section>
						<h2>Large network example</h2>
						<aside class="notes">
						</aside>
					</section>
					<section>
						<h2>Classifying developmental and sensory networks</h2>
						<aside class="notes">
						</aside>
					</section>
				</section>
                
<!-- Large Networks -->

				<section id="largenetworks">
                    
                    <!-- Folie 1 -->
                    
					<section data-background="img/Laura/flagella.jpg" data-state="img-transparent">
						<h2>Large Networks</h2>
					</section>
                    
                    <!-- Folie 2 -->
                    
                    <section>
						<h2>Logic operators</h2>
                        <p>There are different ways how X and Y can be integrated to regulate the Z promoter</p>
                        <span class = "fragment" data-fragment-index= "1"><img  width="134.4" height="150" data-src="img/Laura/Pfeil_unten.png" style="background:none; border:none; box-shadow:none;" >
                        <p>Two common input functions are the <span class = "fragment highlight-blue" data-fragment-index= "2"><strong> AND </strong> </span> gate and <span class = "fragment highlight-green" data-fragment-index= "3"><strong> OR </strong> </span> gate. </p></span>
					</section>
                    
                    <!-- Folie 3 -->
                    
                    <section>
                     
                        <h2>AND Gate</h2>
                        <img width="266" height="569" data-src="img/Laura/and_gate.png" alt="Down arrow"> 
                        <img width="1129" height="569" data-src="img/Laura/and_function.png" alt="Down arrow"> 
                        
                    </section>
                    
                     <!-- Folie 4 -->
                    
                    <section>
                        
                        <h3>OR Gate</h3>
                        <img width="284" height="569" data-src="img/Laura/or_gate.png" alt="Down arrow"> 
                        <img width="1380" height="569" data-src="img/Laura/or_function.png" alt="Down arrow"> 
                        
                    </section>

                        
                    <!-- Folie 5 -->    
<!--
                    <section >
                            <h2>Flagella</h2>
                        
				    </section>
-->
                    
                    <!-- Folie 6 --> 
                    
                    <section>
                        <h2>Detection of network motifs</h2>
                        <p> <small class = "subtitle">Theoretically and Experimentally</small> </p>
                    </section>
                    
                    <!-- Folie 7 --> 
                    
                    <section>
                    <div class = "row">
                        <h2>Finding network motifs theoretically</h2>
                        <div class = "column">
                             <p>Algorithm to detect network motifs:</p>
                            <ol>
                             

                                    <li class = "fragment grow" data-fragment-index= "1"> Create a network with same number of nodes as has the real network</li>
                                    <li class = "fragment grow" data-fragment-index= "2"> Assign the same number of incoming and outgoing edges to each node <br> as has the corresponding node in the real network</li>
                                    <li class = "fragment grow" data-fragment-index= "4"> Scan network for all possible n-node subgraphs and record the number <br> of occurrences of each subgraph </li>
                                    <li class = "fragment grow" data-fragment-index= "5">Compare probabilities &rarr; cut-off probability </li>

                            </ol>
                        </div>
                        <div class = "column">
                               
                            <div>
                                <div class="inner" ><img class = "fragment" data-fragment-index= "1" width="533" height="527" data-src="img/Laura/step1.png" ></div>
                                
                                <div class="inner"><img class = "fragment" data-fragment-index= "2" width="533" height="527" data-src="img/Laura/step2.png" alt="Down arrow"></div>
                                
                                <div class="inner"><img class = "fragment" data-fragment-index= "3" width="533" height="527" data-src="img/Laura/step3.png" alt="Down arrow"></div>
                                <div class="inner"><img class = "fragment" data-fragment-index= "4" width="533" height="527" data-src="img/Laura/random_selfmade.png" alt="Down arrow"></div>
                            
                            </div>

                        </div>
                    </div>
                    </section>
                    
                    <section>
                        <h2>Study network motifs experimentally</h2>
                        <div class = "row">
                            <div class = "column">
                                <ul>
                                	<li class = "fragment grow" data-fragment-index= "1">Measure the transcriptional activity of promoters by means of reporter genes</li>
                                 	<li class = "fragment grow" data-fragment-index= "2">Widely used: Green Fluorescent Protein (GFP) </li>
                                	<li class = "fragment grow" data-fragment-index= "5">Fast-folding GFP variants become fluorescent within minutes after transcription initiation</li>
                                	<li class = "fragment grow" data-fragment-index= "6">Promoter activation is detected by increased fluorescent signal</li>
                                </ul>
                            </div>
                            <div class = "column">
                                
                                <div style = "position: absolute; top:auto; left: 1100px;" ><img class = "fragment current-visible" data-fragment-index= "2" width="803" height="450" data-src="img/Laura/jellyfish.jpeg" style="background:none; border:none; box-shadow:none;" ></div>
                                <div style = "position: absolute; top:auto; left: 1200px;" ><img class = "fragment" data-fragment-index= "3" width="466" height="450" data-src="img/Laura/gene_reporter.png" style="background:none; border:none; box-shadow:none;" ></div>
                                
                                <div style = "position: absolute; top:auto; left: 1100px;" ><img class = "fragment" data-fragment-index= "4" width="754" height="450" data-src="img/Laura/gene_reporter_mRna.png" alt="Down arrow" style="background:none; border:none; box-shadow:none;"></div>
                                
                            </div>
                            
                        </div>
                    </section>
                    </section>
            
<!-- Conclusion & Outlook --> 

				<section id="conclusion">
					<section data-background="img/Laura/cell.jpg" data-state="img-transparent" >
						<h2>Conclusion & Outlook</h2>
					</section>
					<section>
						<div>
                                		<ul>
                                 			<li class = "fragment grow" data-fragment-index= "1" style="height:100px">network motifs are essential to life itself</li>
                                 			<li class = "fragment grow" data-fragment-index= "2" style="height:100px">principle of detecting a signal and generating an appropriate response</li>
                                			<li class = "fragment grow" data-fragment-index= "3" style="height:100px">mathematical representations complement "cartoonish" visualizations</li>
                                			<li class = "fragment grow" data-fragment-index= "4" style="height:100px">accordance with experimental observations</li>
                                			<li class = "fragment grow" data-fragment-index= "4" style="height:100px">breaking down complicated systems/networks into smaller understandable motifs</li>
                                			<li class = "fragment grow" data-fragment-index= "4" style="height:100px">"biological engineering"</li>
                                		</ul>
						</div>
					</section>
				</section>

				<section id="end">
					<h2>Thank you for listening!</h2>
				</section>
                
<!-- Literature -->
                <section>
                <section>
                <div class = "row">
                <h2>Literature</h2>
                    <ul>
                        <li>Alon, Uri. "Network motifs: theory and experimental approaches." <em>Nature Reviews Genetics</em> 8.6 (2007): 450.</li>
                        <li>Tyson, John J., Katherine C. Chen, and Bela Novak. "Sniffers, buzzers, toggles and blinkers: dynamics of regulatory and signaling pathways in the cell." <em>Current opinion in cell biology</em> 15.2 (2003): 221-231.</li>
                        <li>Milo, Ron, et al. "Network motifs: simple building blocks of complex networks." <em>Science</em> 298.5594 (2002): 824-827.</li>
                        <li>Phillips, Rob. "Napoleon is in equilibrium." <em>Annu. Rev. Condens. Matter Phys.</em> 6.1 (2015): 85-111.</li>
                        <li>Kalir, Shiraz, Shmoolik Mangan, and Uri Alon. "A coherent feed‚Äêforward loop with a SUM input function prolongs flagella expression in Escherichia coli." <em>Molecular systems biology</em> 1.1 (2005).</li>
                        <li>Kollmann, Markus, et al. "Design principles of a bacterial signalling network." <em>Nature</em> 438.7067 (2005): 504.</li>
                        <li>Zaslaver, Alon, et al. "A comprehensive library of fluorescent transcriptional reporters for Escherichia coli." <em>Nature methods</em> 3.8 (2006): 623.</li>
                        <li>Uri Alon. "Network Motifs." URL: https://www.weizmann.ac.il/mcb/UriAlon/research/network-motifs (19.06.2018) </li>
                    </ul>
                </div>
                </section>
                    
            
                
                
<!-- Pictures -->
                
                <section>
                    <h2>Pictures</h2>
                    <div class = "row">
                        <div class = "column">
                        
                           <img width="400" height="314" data-src="img/Laura/trans_trans.png" style="background:none; border:none; box-shadow:none;" > 
                            <img  data-fragment-index = "0" width="500" height="250" data-src="img/Laura/activator.png" style="background:none; border:none; box-shadow:none;" >
                            <img  width="974" height="250" data-src="img/Laura/repressor.png" style="background:none; border:none; box-shadow:none;" >
                        
                        </div>
                        <div style = "position:absolute; left:1000px; top:350px" >
                            <p>Phillips, Rob. "Napoleon is in equilibrium." <em>Annu. Rev. Condens. Matter Phys.</em> 6.1 (2015): 85-111.</p>
                        </div>
                    
                    
                    </div>
                
                </section>
                <section>
                    <div class = "row">
                        <div class = "column">
                            
                        </div>
                    </div>
                </section>
                
                </section>

			</div>

		</div>

		<script src="lib/js/head.min.js"></script>
		<script src="js/reveal.js"></script>

		<script>

			// More info https://github.com/hakimel/reveal.js#configuration
			Reveal.initialize({
				controls: true,
				progress: true,
				history: true,
				center: true,

				slideNumber: true,
				width: 1920,
				height: 1080,
				viewDistance: 3,

				transition: 'slide', // none/fade/slide/convex/concave/zoom

				// More info https://github.com/hakimel/reveal.js#dependencies
				dependencies: [
					{ src: 'lib/js/classList.js', condition: function() { return !document.body.classList; } },
					{ src: 'plugin/markdown/marked.js', condition: function() { return !!document.querySelector( '[data-markdown]' ); } },
					{ src: 'plugin/markdown/markdown.js', condition: function() { return !!document.querySelector( '[data-markdown]' ); } },
					{ src: 'plugin/highlight/highlight.js', async: true, callback: function() { hljs.initHighlightingOnLoad(); } },
					{ src: 'plugin/search/search.js', async: true },
					{ src: 'plugin/zoom-js/zoom.js', async: true },
					{ src: 'plugin/notes/notes.js', async: true }
				]
			});
			
			// Slide number formatting can be configured using these variables:
			//  "h.v": 	horizontal . vertical slide number (default)
			//  "h/v": 	horizontal / vertical slide number
			//    "c": 	flattened slide number
			//  "c/t": 	flattened slide number / total slides
			Reveal.configure({ slideNumber: 'c' });
			
			// Control which views the slide number displays on using the "showSlideNumber" value:
			//     "all": show on all views (default)
			// "speaker": only show slide numbers on speaker notes view
			//   "print": only show slide numbers when printing to PDF
			//Reveal.configure({ showSlideNumber: 'speaker' });

		</script>

	</body>
</html>
