<!doctype html>
<html lang="en">

	<head>
		<meta charset="utf-8">

		<title>Network Motifs</title>

		<meta name="description" content="Understanding Biology Through Mathematics">
		<meta name="author" content="Laura Martens & Arthur Heimbrecht">

		<meta name="apple-mobile-web-app-capable" content="yes">
		<meta name="apple-mobile-web-app-status-bar-style" content="black-translucent">

		<meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=no">

		<link rel="stylesheet" href="css/reveal.css">
		<link rel="stylesheet" href="css/theme/moon.css" id="theme">

		<!-- Theme used for syntax highlighting of code -->
		<link rel="stylesheet" href="lib/css/zenburn.css">

		<!-- Printing and PDF exports -->
		<script>
			var link = document.createElement( 'link' );
			link.rel = 'stylesheet';
			link.type = 'text/css';
			link.href = window.location.search.match( /print-pdf/gi ) ? 'css/print/pdf.css' : 'css/print/paper.css';
			document.getElementsByTagName( 'head' )[0].appendChild( link );
		</script>
        <style type="text/css">
            .inner {
              position: absolute; right: 100px; bottom: -30px
            }
            .inner_2 {
              position: absolute; margin: 0 auto;
            }

        </style>

		<!--[if lt IE 9]>
		<script src="lib/js/html5shiv.js"></script>
		<![endif]-->
	</head>

	<body>

		<div class="reveal">

			<!-- Any section element inside of this container is displayed as a slide -->
			<div class="slides">
				<section data-background="img/network3.png" data-state="img-transparent" id="title">
					<div class="bodyneeded">
						<h1>Network Motifs</h1>
						<h3>Understanding Biology through Mathematics</h3>
						<p>
							<small class="subtitle">presented by Laura Martens & Arthur Heimbrecht</small>
						</p>
					</div>
				</section>

                
<!--Contents-->     
                
				<section >
					<h2>Contents</h2>
                    <p>Introduction</p>
                    <p>Mathematical Models</p>
                    <p>Large Networks</p>
                    <p>Conclusion & Outlook</p>
                    
					<aside class="notes">
						1. Anwer question What are network motifs 
                        2 Arthur: Mathematically model network motifs and their function
                        3. Large networks, how to detect network motifs
                        4. Finish with..
                        
                        Lets start off with the question ..
					</aside>
				</section>

<!--Introduction-->                
                
				<section id="introduction">
                    
                    
                    <!-- Folie 1 -->
					<section data-background="img/Laura/cell.jpg" data-state="img-transparent">
						<h2>Introduction</h2>
					</section>
                    
                    <!-- Folie 2 -->
					<section data-transition="slide">
						
							<h2>What are network motifs?</h2>
                            
								 <ul style = "position:relative; margin-left: 100px; margin-right: 100px;"> 
                                    <li> <p class="fragment">Basic interaction patterns that recur throughout biological and non-biological networks much more often than in random networks</p></li> 
                                    <li class = "fragment"> <p>Small set of network motifs serve as bulding blocks of large networks</p> </li>
                                    <li class = "fragment"> <p> Each network has a defined function: e.g. filters, pulse generators</p>  </li> 
                                    
                                 </ul> 
                            
                                <p class="fragment"> <img width="715" height="400" data-src="img/Laura/motif2.png" style="background:none; border:none; box-shadow:none;" > </p>
                            
						
                        <aside class = "notes">
                            
                            1. basic interaction pattern
                            network motifs form... &rArr; defined function
                            
                            Interestingly there is only a small set of recurring network motifs serve as building blocks 
                            
                            Picture: one of the most common network motifs - FFL 
                                    interaction between nodes are illustrated by directed egdes
                                    1 regulates two where the second one also controls third..
                            
                            so this network motif was detected by ..
                                
                        
                        </aside>
					</section>
                    
                    <!-- Folie 2 continued-->
                    
                    <section data-transition="slide">
						
                        <div class="row">
                            
                            <div class="column" class = "fragment fade-up" >	 
                                <h2 > Real network</h2>
                                <br>
                                <img width="533" height="527" data-src="img/Laura/real_selfmade.png" alt="Down arrow">
                                
                                <img class = "fragment" width="200" height="184" data-src="img/Laura/Pfeil.png" style="background:none; border:none; box-shadow:none; position: absolute; left:860px; top:350px" data-fragment-index= "2">
                            </div>
                            
                            <div class="column" class = "fragment" data-fragment-index= "1">	 
                                <h2> Random network</h2>
                                <br>
                                <img width="533" height="527" data-src="img/Laura/random_selfmade.png" alt="Down arrow">      
                            </div>
                            
                            
						</div>
                        
                        <aside class = "notes">
                            1. Compare the two networks
                            in general a network does not consist of only three nodes...
                        </aside>
					</section>
                    
                    <!-- Folie 3 -->
<!--
					<section>
                       
						<div >
							<h2>Network motifs as building blocks</h2> 
                    
                            
                       
                           <ul>
                                
                               <li class = "fragment" data-fragment-index= "1"> <p> Each network has a defined function: e.g. filters, pulse generators </p>  </li> 
                            <li class = "fragment" data-fragment-index= "2"> <p>Small set of network motifs serve as bulding blocks of large networks</p> </li>
                            </ul> 

                            
                       
                            
                        </div> 
                        
                        <aside class = "notes">
                            network motifs form... &rArr; defined function
                            
                            Interestingly there is only a small set of recurring network motifs serve as building blocks 
                        </aside>
                            
					</section>
-->
                    
                    <section>
                    
                         <div style = "position:absolute; top:20px;">
                            <img class = "fragment grow" data-fragment-index = "1" width="885" height="500" data-src="img/Laura/transcription_network.png" style="background:none; border:none; box-shadow:none;" >
                            <img class = "fragment shrink" data-fragment-index = "1" width="885" height="500" data-src="img/Laura/neurons.png" style="background:none; border:none; box-shadow:none;" >
                            <img class = "fragment shrink" data-fragment-index = "1" width="885" height="500" data-src="img/Laura/www.png" style="background:none; border:none; box-shadow:none; " >
                        </div>
                        
                        <aside class = "notes">
                        
                                a few examples: 
                            
                            1. Transcription regulation networks &rArr; biochemical networks responsible
                                for regulating the expression of genes in cells
                            2. Nodes represent neurons (or neuronclasses), and edges represent synaptic connections
                                between the neurons.
                            (  This similarity in motifs &rArr;similarity in the design constraints of the two types of network )
                            
                            3. a network of directed hyperlinks between World Wide Web pages within a single domain The World Wide Web motifs may reflect a design aimed at short paths between related pages.
                            
                            As the network motifs in transcripton networks are the most studied so far &rArr; Refresh your memory of how they work
                            
                        </aside>
                    </section>

                    
                     <!-- Folie 4 -->
                    
                    <section>
						
							<h2>Transcription networks</h2> 
                            <img width="952" height="750" data-src="img/Laura/trans_trans.png" style="background:none; border:none; box-shadow:none;" >

						<aside class = "notes">
                            In the production of proteins there are two important stages: blabla
                            
                            this process can either be accelrated or inhibited by transcription factor proteins
                        </aside>
					</section>
                    
                    <section>
                        
                        <h2>Activators and repressors</h2>
                        <div style = "position: absolute; left: 450px; bottom:auto"><img class = "fragment current-visible" data-fragment-index = "0" width="996" height="525" data-src="img/Laura/activator.png" style="background:none; border:none; box-shadow:none;" ></div>
                        <div style = "position: absolute; left: 180px;"><img class = "fragment" data-fragment-index = "1" width="1572" height="400" data-src="img/Laura/repressor.png" style="background:none; border:none; box-shadow:none;" ></div>
                        
                        
                        <aside class = "notes">
                            blabla
                            as the transcripion proteins themselve arise from a transcription -translation process it can form interesting dynamics in the form of a transription regulation network. 
                            They can mathematically described by setting up differential equation with the reaction rates k as Arthur..
                            
                            We have seen that in biologcical networks (Trans and neurons) the same network, which gives rise to the question how these network motifs have evolved
                        </aside>
                    </section>
                    
                   
                    
                    <!-- Folie 5 -->
                    <section><h2>Evolution of network motifs</h2></section>
                    <section>

                            <div class = "inner_2"><img  width="1810" height="980" data-src="img/Laura/Evolution_1.png" style="background:none; border:none; box-shadow:none;" ></div>
                            <div class = "inner_2"><img class = "fragment"  width="1810" height="980" data-src="img/Laura/Evolution_2.png" style="background:none; border:none; box-shadow:none;" ></div>
                            <div class = "inner_2"><img class = "fragment"  width="1810" height="980" data-src="img/Laura/Evolution_3.png" style="background:none; border:none; box-shadow:none;" ></div>
                            <div class = "inner_2"><img class = "fragment"  width="1810" height="980" data-src="img/Laura/Evolution_4.png" style="background:none; border:none; box-shadow:none;" ></div>
                            <div class = "inner_2"><img class = "fragment"  width="1810" height="980" data-src="img/Laura/Evolution.png" style="background:none; border:none; box-shadow:none;" ></div>
                    <aside class = "notes"> 
                         1.First of all.. Biochemical networks noisy environment, imperfect components
                        
                        2. Even though the conditionns might flctuate the tolerance in the output is very low
                        
                        3. Erros can lead to wrong decisions or development &rArr; could be fatal
                        
                        4. Strong selective pressure against robust networks that work in unstable environment pertubations eg. fluctuations in protein concentration
                        
                        5. Has been shown that the evolved network motif are the most robust networks with the least number of nodes at the same time
                        
                        
                    </aside>
                    

                    </section>
                    
                    
                    

				</section>

<!--Mathematical models-->   
                
				<section id="biology">
					<section data-background="img/Laura/dna.jpg" data-state="img-transparent" >
						<h2>Mathematical models</h2>
<!--						<p>Buzzers, Sniffers, Switches, Blinkers</p>-->
						<aside class="notes">
							<ul>
								<li>recognised so of Lauras topics, should apply here as well</li>
								<li>explain mathematics step by step</li>
								<li>QUESTIONS</li>
							</ul>
						</aside>
					</section>
					<section>
						<h2>Classifying developmental and sensory networks</h2>
                                		<ul>
							<li class = "fragment" data-fragment-index= "1" style="height:100px"><b>Sensory networks</b></li>
                                			<ul>
                                 				<li class = "fragment" data-fragment-index= "2" style="height:100px">Cell "senses" its surroundings/its own state</li>
                                				<li class = "fragment" data-fragment-index= "3" style="height:100px">Response is a function of the signal</li>
                                			</ul>
							<li class = "fragment" data-fragment-index= "4" style="height:100px"><b>Developmental networks</b></li>
                                			<ul>
                                				<li class = "fragment" data-fragment-index= "5" style="height:100px">Response does not depend on the signal if threshold is reached</li>
                                 				<li class = "fragment" data-fragment-index= "6" style="height:100px">Cell fate decisions are final</li>
                                			</ul>
                                		</ul>
						<aside class="notes">
							<ul>
								<li>how to distinguish netowrk motifs</li>
								<li>featured by Uri Alon</li>
								<li>Can you answer which network is which type?</li>
								<li>sensory alsways takes signal into account</li>
								<li>developmental is independent of S if threshold is reached</li>
								<li>irrevesible decision like apoptosis</li>
							</ul>
						</aside>
					</section>
					<section>
						<h3>Preliminary example</h3>
							<div class="row" style="background-color:#FFF;">
								<div class="column">
									<img  width="800" data-src="img/wire1a.png" style="background:none; border:none; box-shadow:none;" >
								</div>
								<div class="column">
									<img  width="800" data-src="img/diff1a.png" style="background:none; border:none; box-shadow:none;" >
								</div>
							</div>
						<aside class="notes">
							<ul>
								<li>2 quantities</li>
								<li>arrows = wires</li>
								<li>solid = current</li>
								<li>dashed = amplification</li>
								<li>ODE w/ law of mass action</li>
								<li>aggregation blue/depeltion red</li>
								<li>dependencies</li>
								<li>long term solution -> dR/dt = 0</li>
								<li>steady state</li>
								<li>SSS is easy to find</li>
							</ul>
						</aside>
							
					</section>
					<section>
						<h3>Preliminary example</h3>
						<iframe data-src="http://localhost:5006/diagram1a", width=1550, height=850px style="background: #FFFFFF;"></iframe>
						<aside class="notes">
							<ul>
								<li>left is aggregation and depletion on dR/dt over R</li>
								<li>right is signal response plot</li>
								<li>aggregation independent of R but SLIDER</li>
								<li>depletion depends only on R</li>
								<li>intersection of graphs is SSS</li>
								<li>right plot shows same steady state</li>
							</ul>
						</aside>
					</section>
					<section>
						<h3>Biological example</h3>
							<div class="row" style="background-color:#FFF;">
								<div class="column">
									<img  width="800" data-src="img/wire1b.png" style="background:none; border:none; box-shadow:none;" >
								</div>
								<div class="column">
									<img  width="800" data-src="img/diff1b.png" style="background:none; border:none; box-shadow:none;" >
								</div>
							</div>
						<aside class="notes">
							<ul>
								<li>no time so jump to Michaelis Menten kinetics</li>
								<li>seen this yesterday</li>
							</ul>
						</aside>
							
					</section>
					<section>
						<h3>Biological example</h3>
						<iframe data-src="http://localhost:5006/diagram1b", width=1550, height=850px style="background: #FFFFFF;"></iframe>
					</section>
					<section>
						<h3>Buzzer</h3>
							<div class="row" style="background-color:#FFF;">
								<div class="column">
									<img  width="600" data-src="img/wire1c.png" style="background:none; border:none; box-shadow:none;" >
								</div>
								<div class="column">
									<img  width="1200" data-src="img/diff1c.png" style="background:none; border:none; box-shadow:none;" >
								</div>
							</div>
						<aside class="notes">
							<ul>
								<li>phosphorylation of R controlled by S</li>
								<li>degradation of RP with H2O</li>
								<li>RP != R</li>
								<li>aggr and depl follows MM kinetics</li>
								<li>longterm solution can be calculated</li>
								<li>Goldbeter-Koshland function</li>
							</ul>
						</aside>
							
					</section>
					<section>
						<h3>Buzzer</h3>
						<iframe data-src="http://localhost:5006/diagram1c", width=1550, height=850px style="background: #FFFFFF;"></iframe>
						<aside class="notes">
							<ul>
								<li>Goldbeter Koshland</li>
								<li>sigmoidal shape!!!</li>
								<li>cahnge S and look</li>
								<li>sigmoidal shape!!</li>
								<li>zero order ultrasensitivity</li>
								<li>buzzer from doorbell</li>
							</ul>
						</aside>
					</section>
					<section>
						<h3>Sniffer</h3>
							<div class="row" style="background-color:#FFF;">
								<div class="column">
									<img  width="800" data-src="img/wire1d.png" style="background:none; border:none; box-shadow:none;" >
								</div>
								<div class="column">
									<img  width="800" data-src="img/diff1d.png" style="background:none; border:none; box-shadow:none;" >
								</div>
							</div>
						<aside class="notes">
							<ul>
								<li>olfactory sense</li>
								<li>new quantity X</li>
								<li>S amplifies both</li>
								<li>X amplifies degradation</li>
								<li>differential equation</li>
								<li>solve for Xss</li>
								<li>R has SSS independent of S!?!?!?!</li>
							</ul>
						</aside>
							
					</section>
					<section>
						<h3>Sniffer</h3>
						<iframe data-src="http://localhost:5006/diagram1d", width=1550, height=850px style="background: #FFFFFF;"></iframe>
						<aside class="notes">
							<ul>
								<li>similar to first example</li>
								<li>change S slider</li>
								<li>R at intersection constant</li>
								<li>perfect adaption</li>
								<li>chemotactic systems</li>
								<li>sniffer since smelling adaption over time</li>
								<li>time is ignored</li>
							</ul>
						</aside>
					</section>
					<section>
						<h3>Sniffer</h3>
							<div class="row" style="background-color:#FFF;">
								<div class="column">
									<img  width="800" data-src="img/adaption.png" style="background:none; border:none; box-shadow:none;" >
								</div>
								<div class="column">
									<img  width="800" data-src="img/diff1d.png" style="background:none; border:none; box-shadow:none;" >
								</div>
							</div>
						<aside class="notes">
							<ul>
								<li>X slowly reaches its SSS</li>
								<li>time delay</li>
								<li>R has peak and adapts</li>
								<li>size of peaks</li>
								<li>overdamped???????</li>
							</ul>
						</aside>
							
					</section>
					<section>
						<h3>Irreversible switch</h3>
							<div class="row" style="background-color:#FFF;">
								<div class="column">
									<img  width="800" data-src="img/wire1e.png" style="background:none; border:none; box-shadow:none;" >
								</div>
								<div class="column">
									<img  width="800" data-src="img/diff1e.png" style="background:none; border:none; box-shadow:none;" >
								</div>
							</div>
						<aside class="notes">
							<ul>
								<li>positive feedback loop</li>
								<li>R amplifies E to EP, seen this already</li>
								<li>SSS of phosphorylation</li>
							</ul>
						</aside>
					</section>
					<section>
						<h3>Irreversible switch</h3>
						<iframe data-src="http://localhost:5006/diagram1e", width=1550, height=850px style="background: #FFFFFF;"></iframe>
						<aside class="notes">
							<ul>
								<li>aggregation is sigmoidal</li>
								<li>three solutions!!</li>
								<li>first one is stable</li>
								<li>second one is unstable</li>
								<li>signal response curve</li>
								<li>low signal low response</li>
								<li>critical S, bifurcation, jump</li>
								<li>upper steady state is final</li>
								<li>cannot come back down like cat in tree</li>
								<li>mutual activation</li>
								<li>cell fate decisions</li>
							</ul>
						</aside>
						
					</section>
					<section>
						<h3>Reversible switch</h3>
							<div class="row" style="background-color:#FFF;">
								<div class="column">
									<img  width="800" data-src="img/wire1f.png" style="background:none; border:none; box-shadow:none;" >
								</div>
								<div class="column">
									<img  width="800" data-src="img/diff1f.png" style="background:none; border:none; box-shadow:none;" >
								</div>
							</div>
						<aside class="notes">
							<ul>
								<li>change system a bit</li>
								<li>mutually antagonistic</li>
								<li>depletion is amplified</li>
							</ul>
						</aside>
							
					</section>
					<section>
						<h3>Reversible switch</h3>
						<iframe data-src="http://localhost:5006/diagram1f", width=1550, height=850px style="background: #FFFFFF;"></iframe>
						<aside class="notes">
							<ul>
								<li>3 solutions</li>
								<li>1 solution on each side</li>
								<li>jump back</li>
								<li>mitosis</li>
							</ul>
						</aside>
					</section>
					<section>
						<h3>Positive feedback oscillator</h3>
							<div class="row" style="background-color:#FFF;">
								<div class="column">
									<img  width="800" data-src="img/wire2b.png" style="background:none; border:none; box-shadow:none;" >
								</div>
								<div class="column">
									<img  width="800" data-src="img/diff2b.png" style="background:none; border:none; box-shadow:none;" >
								</div>
							</div>
						<aside class="notes">
							<ul>
								<li>skip Blinker since not intuitive and this is biol. relevant</li>
								<li>toggle switch + </li>
								<li>similar to perfect adaption</li>
							</ul>
						</aside>
							
					</section>
					<section>
						<h3>Positive feedback oscillator</h3>
						<iframe data-src="http://localhost:5006/diagram2b", width=1550, height=850px style="background: #FFFFFF;"></iframe>
						<aside class="notes">
							<ul>
								<li>no rates but X-R plane!!!1</li>
								<li>nullclines</li>
								<li>low S stable SSS</li>
								<li>medium S instable solution for R</li>
								<li>oscialltion in stable orbit</li>
								<li>Hopf bifurcation</li>
								<li>larger S stable solution</li>
								<li>example: circadian rhythm and call maturation</li>
							</ul>
						</aside>
						
					</section>
					<section>
						<h3>Negative feedback oscillator</h3>
							<div class="row" style="background-color:#FFF;">
								<div class="column">
									<img  width="800" data-src="img/wire2c.png" style="background:none; border:none; box-shadow:none;" >
								</div>
								<div class="column">
									<img  width="800" data-src="img/diff2c.png" style="background:none; border:none; box-shadow:none;" >
								</div>
							</div>
						<aside class="notes">
							<ul>
								<li>no time to explain</li>
							</ul>
						</aside>
							
					</section>
					<section>
						<h3>Negative feedback oscillator</h3>
						<iframe data-src="http://localhost:5006/diagram2c", width=1550, height=850px style="background: #FFFFFF;"></iframe>
						<aside class="notes">
							<ul>
								<li>improvise</li>
							</ul>
						</aside>
					</section>
					<section>
						<h2>Large network example</h2>
						<img  height="800" data-src="img/graph.png" style="background:none; border:none; box-shadow:none;" >
						<aside class="notes">
							<ul>
								<li>cell development, 3 phases</li>
								<li>looks complicated but seen in abstract form</li>
							</ul>
						</aside>
						
					</section>
					<section>
						<h2>Large network example</h2>
						<img  height="800" data-src="img/response.png" style="background:none; border:none; box-shadow:none;" >
						<aside class="notes">
							<ul>
								<li>cell 'birth', small size</li>
								<li>jump to toggle switch</li>
								<li>maturation into oscillation</li>
								<li>infinite period</li>
								<li>when back down mitosis</li>
								<li>start again</li>
								<li>what is Cdk1 CycB</li>
							</ul>
						</aside>
						
					</section>
				</section>
                
<!-- Large Networks -->

				<section id="largenetworks">
                    
                    <!-- Folie 1 -->
                    
					<section data-background="img/Laura/flagella.jpg" data-state="img-transparent">
						<h2>Large Networks</h2>
                        
                        <aside class = "notes"> As i said befor e network motifs are building blocks of bigger networks
                            
                        But not only the network motif itself has a defined function, but the behaviour can also alter by differetn inout functions the so called logic operators: 
                        
                        
                        </aside>
					</section>
                    
                    <!-- Folie 2 -->
                    
                    <section>
						<h2>Logic operators</h2>
                        <p>There are different ways how X and Y can be integrated to regulate the Z promoter</p>
                        <span class = "fragment" data-fragment-index= "1"><img  width="134.4" height="150" data-src="img/Laura/Pfeil_unten.png" style="background:none; border:none; box-shadow:none;" >
                        <p>Two common input functions are the <span class = "fragment highlight-blue" data-fragment-index= "2"><strong> AND </strong> </span> gate and <span class = "fragment highlight-blue" data-fragment-index= "3"><strong> OR </strong> </span> gate. </p></span>
                        
                        <aside class = "notes">
                            two common .. bla bla
                            
                            were biding of both regulators is necessary .. 
                        </aside>
					</section>
                    
                    <!-- Folie 3 -->
                    
                    <section>
                     
                        <h2>AND Gate</h2>
                        <img width="266" height="569" data-src="img/Laura/and_gate.png" alt="Down arrow"> 
                        <img width="1129" height="569" data-src="img/Laura/and_function.png" alt="Down arrow"> 
                        
                    </section>
                    
                     <!-- Folie 4 -->
                    
                    <section>
                        
                        <h3>OR Gate</h3>
                        <img width="284" height="569" data-src="img/Laura/or_gate.png" alt="Down arrow"> 
                        <img width="1380" height="569" data-src="img/Laura/or_function.png" alt="Down arrow"> 
                        
                        <aside class = "notes">
                          Flaggela system of E.Coli 
                            when conditions are unfavourable glucose starvation, osmotic stress etc. E coli produces flagella and swims away 
                            
                            
                            has a FFl with or logic. 
                            
                            The reason becomes clear when looking at the graph &rArr; Sx are factors that are controlled by factors that respond to environmental signals 
                            
                            delay even though signal isn't there
                            
                            ecoli only stops when conditions have been good for a longer time
                            
                            
                            
                        
                        </aside>
                        
                    </section>

                        
                    <!-- Folie 5 -->    
<!--
                    <section >
                            <h2>Flagella</h2>
                        
				    </section>
-->
                    
                    <!-- Folie 6 --> 
                    
                    <section>
                        <h2>Detection of network motifs</h2>
                        <p> <small class = "subtitle">Theoretically and Experimentally</small> </p>
                    </section>
                    
                    <!-- Folie 7 --> 
                    
                    <section>
                    <div class = "row">
                        <h2>Finding network motifs theoretically</h2>
                        <div class = "column2">
                             <p>Algorithm to detect network motifs:</p>
                            <ul>
                             

                                    <li class = "fragment grow" data-fragment-index= "1"> Create a network with same number of nodes as the real network</li>
                                    <li class = "fragment grow" data-fragment-index= "2"> Assign the same number of incoming and outgoing edges to each node <br> as has the corresponding node in the real network</li>
                                    <li class = "fragment grow" data-fragment-index= "4"> Scan network for all possible n-node subgraphs and record the number <br> of occurrences of each subgraph </li>
                                    <li class = "fragment grow" data-fragment-index= "5">Compare probabilities &rarr; cut-off probability </li>

                            </ul>
                        </div>
                        <div class = "column">
                               
                            <div>
                                <div class="inner" ><img class = "fragment" data-fragment-index= "1" width="533" height="527" data-src="img/Laura/step1.png" ></div>
                                
                                <div class="inner"><img class = "fragment" data-fragment-index= "2" width="533" height="527" data-src="img/Laura/step2.png" alt="Down arrow"></div>
                                
                                <div class="inner"><img class = "fragment" data-fragment-index= "3" width="533" height="527" data-src="img/Laura/step3.png" alt="Down arrow"></div>
                                <div class="inner"><img class = "fragment" data-fragment-index= "4" width="533" height="527" data-src="img/Laura/random_selfmade.png" alt="Down arrow"></div>
                            
                            </div>

                        </div>
                    </div>
                    </section>
                    
                    <section>
                        <h2>Study network motifs experimentally</h2>
                        <div class = "row">
                            <div class = "column2">
                                <ul>

                                 <li class = "fragment grow" data-fragment-index= "1" >Measure the transcriptional activity of promoters by means of reporter genes</li>
                                 <li class = "fragment grow" data-fragment-index= "2">Widely used: Green Fluorescent Protein (GFP) </li>
                                <li class = "fragment grow" data-fragment-index= "5">Promoter activation is detected by increased fluorescent signal</li>
                                <li class = "fragment grow" data-fragment-index= "6">Fast-folding GFP variants become fluorescent within minutes after transcription initiation</li>

                                </ul>
                            </div>
                            <div class = "column">
                                
                                <div class = "inner" ><img class = "fragment current-visible" data-fragment-index= "2" width="625" height="350" data-src="img/Laura/jellyfish.jpeg" style="background:none; border:none; box-shadow:none;" ></div>
                                <div class = "inner" ><img class = "fragment" data-fragment-index= "3" width="362" height="350" data-src="img/Laura/gene_reporter.png" style="background:none; border:none; box-shadow:none;" ></div>
                                
                                <div class = "inner"><img class = "fragment" data-fragment-index= "4" width="586" height="350" data-src="img/Laura/gene_reporter_mRna.png" alt="Down arrow" style="background:none; border:none; box-shadow:none;"></div>
                                
                            </div>
                            
                        </div>
                        
                        <aside class = "notes">
                            
                            1.Green flourescent protein: firtly discovered in a type of jellyfish. 
                            gene is inserted after the promoter of the gene of interest.
                            2.fusion protein indicates active promoter 
                            3. fast folfing &rArr; high temporal resolution measurements of promoter activity 
                            
                            This is mehtod how flagella system in Ecoli was studied
                            
                            
                            
                            TESTESTETSTSTS
                        </aside>
                    </section>
                    </section>
            
<!-- Conclusion & Outlook --> 

				<section id="conclusion">
					<section data-background="img/Laura/cell.jpg" data-state="img-transparent" >
						<h2>Conclusion</h2>

                        
				</section>
                    
                    
                    
                    
                    <section>
                        
                     <div class = "row">
                         <h2>Conclusion & Outlook </h2>
                      <ul>
                          <li class = "fragment grow" style="height:100px">Complex biological networks contain a limited set of network motifs</li>
                          <li class = "fragment grow" style="height:100px">
                             Networks can be broken down into smaller understandable motifs</li>
                          <li class = "fragment grow" style="height:100px">Certain network motifs have been selected again and again in evolution</li>
                        <li class = "fragment grow" style="height:100px">Mathematical modeling has helped to define functions of network motifs</li>
                          
                          
                          <li class = "fragment grow" style="height:100px">How do network motifs work in large networks?</li>
                          <li class = "fragment grow" style="height:100px">Further research on other networks</li>
                          
                        </ul>
                         
                         </div>
                    </section>

<!--
					
					<section>
						<div>
                                		<ul>
                                 			<li class = "fragment grow" data-fragment-index= "1" style="height:100px">network motifs are essential to life itself</li>
                                 			<li class = "fragment grow" data-fragment-index= "2" style="height:100px">principle of detecting a signal and generating an appropriate response</li>
                                			<li class = "fragment grow" data-fragment-index= "3" style="height:100px">mathematical representations complement "cartoonish" visualizations</li>
                                			<li class = "fragment grow" data-fragment-index= "4" style="height:100px">accordance with experimental observations</li>
                                			<li class = "fragment grow" data-fragment-index= "4" style="height:100px">breaking down complicated systems/networks into smaller understandable motifs</li>
                                		</ul>
						</div>
					</section>
-->

				</section>

				<section id="end">
					<h2>Thank you for listening!</h2>
				</section>
                
<!-- Literature -->
                <section>
                <section>
                <div class = "row">
                <h2>Literature</h2>
                    <ul>
                        <li>Alon, Uri. "Network motifs: theory and experimental approaches." <em>Nature Reviews Genetics</em> 8.6 (2007): 450.</li>
                        <li>Tyson, John J., Katherine C. Chen, and Bela Novak. "Sniffers, buzzers, toggles and blinkers: dynamics of regulatory and signaling pathways in the cell." <em>Current opinion in cell biology</em> 15.2 (2003): 221-231.</li>
                        <li>Milo, Ron, et al. "Network motifs: simple building blocks of complex networks." <em>Science</em> 298.5594 (2002): 824-827.</li>
                        <li>Phillips, Rob. "Napoleon is in equilibrium." <em>Annu. Rev. Condens. Matter Phys.</em> 6.1 (2015): 85-111.</li>
                        <li>Kalir, Shiraz, Shmoolik Mangan, and Uri Alon. "A coherent feed‐forward loop with a SUM input function prolongs flagella expression in Escherichia coli." <em>Molecular systems biology</em> 1.1 (2005).</li>
                        <li>Kollmann, Markus, et al. "Design principles of a bacterial signalling network." <em>Nature</em> 438.7067 (2005): 504.</li>
                        <li>Zaslaver, Alon, et al. "A comprehensive library of fluorescent transcriptional reporters for Escherichia coli." <em>Nature methods</em> 3.8 (2006): 623.</li>
                        <li>Uri Alon. "Network Motifs." URL: https://www.weizmann.ac.il/mcb/UriAlon/research/network-motifs (19.06.2018) </li>
                    </ul>
                </div>
                </section>
                    
            
                
                
<!-- Pictures -->
                
                <section>
                    <h2>Pictures</h2>
                    <div class = "row">
                        <div class = "column">
                        
                           <img width="400" height="314" data-src="img/Laura/trans_trans.png" style="background:none; border:none; box-shadow:none;" > 
                            <img  data-fragment-index = "0" width="500" height="250" data-src="img/Laura/activator.png" style="background:none; border:none; box-shadow:none;" >
                            <img  width="974" height="250" data-src="img/Laura/repressor.png" style="background:none; border:none; box-shadow:none;" >
                        
                        </div>
                        <div style = "position:absolute; left:1000px; top:350px" >
                            Phillips, Rob. "Napoleon is in equilibrium." <em>Annu. Rev. Condens. Matter Phys.</em> 6.1 (2015): 85-111.
                        </div>
                    
                    
                    </div>
                
                </section>
                    
    
                <section>
                    <div class = "row">
                        <div class = "column">
                        <img  height="285" data-src="img/Laura/and_gate.png" alt="Down arrow"> 
                        <img  height="285" data-src="img/Laura/and_function.png" alt="Down arrow"><br> 
                        <img  height="285" data-src="img/Laura/or_gate.png" alt="Down arrow"> 
                        <img height="285" data-src="img/Laura/or_function.png" alt="Down arrow"> 
                        </div>
                        
                        <div style = "position:absolute; left:1000px; top:350px">
                         Alon, Uri. "Network motifs: theory and experimental approaches." <em>Nature Reviews Genetics</em> 8.6 (2007): 450
                        </div>
                    </div>
                    
                </section>
                <section>
                  <div class = "row">
                    <div class= "column">
                        <img height = "200" data-src = "img/Laura/jellyfish.jpeg"><br>
                        <img height = "200" data-src = "img/Laura/gene_reporter.png"><br>
                        <img height = "200" data-src = "img/Laura/gene_reporter_mRNA.png">
                        
                        
                        
                      </div>
                      <div class = "column">
                        <li style="margin-top:50px; margin-bottom: 50px;">https://www.gettyimages.de/-video/green-jellyfish?sort=most-popular&offlinecontent=include&phrase=green-%20jellyfish </li> 
                        <li style="margin-top:200px; margin-bottom: 200px;">http://www.dnaftb.org/34/problem.html</li> 
                        
                        
                      </div>
                    
                    </div>    
                </section>
                    
                <section>
                  <div class = "row">
                    <div class= "column">
                        <img height = "200" data-src = "img/network3.png"><br>
                        <img height = "200" data-src = "img/Laura/cell.jpg"><br>
                        <img height = "200" data-src = "img/Laura/dna.jpg"><br>
                        <img height = "200" data-src = "img/Laura/flagella.jpg">
                        
                        
                      </div>
                      <div class = "column">
                        <li style="margin-top:100px; margin-bottom: 150px;">http://tracks.arte.tv/fr/surveillance-deni-de-facies</li> 
                        <li style="margin-top:100px; margin-bottom: 150px;">https://www.utahpeoples-post.com/2016/10/human-cell-atlas/</li> 
                        <li style="margin-top:100px; margin-bottom: 150px;">http://tracks.arte.tv/fr/surveillance-deni-de-facies </li>
                        <li style="margin-top:100px; ">https://www.sciencenewsfor-students.org/article/bacteria-help-water-effortlessly-go-flow</li>
                      </div>
                    
                    </div>    
                </section>
                
                </section>
<!-- BackUP -->
            <section>
               <p > <img height="700"  data-src="img/Laura/Numbers.png" style="background:none; border:none; box-shadow:none;" > </p>
            </section>
            <section>
                <p > <img width="1000"  data-src="img/Laura/Flagella_system.png" style="background:none; border:none; box-shadow:none;" > </p>
            </section>


		<script src="lib/js/head.min.js"></script>
		<script src="js/reveal.js"></script>

		<script>

			// More info https://github.com/hakimel/reveal.js#configuration
			Reveal.initialize({
				controls: true,
				progress: true,
				history: true,
				center: true,

				slideNumber: true,
				width: 1920,
				height: 1080,
				viewDistance: 3,

				transition: 'slide', // none/fade/slide/convex/concave/zoom

				// More info https://github.com/hakimel/reveal.js#dependencies
				dependencies: [
					{ src: 'lib/js/classList.js', condition: function() { return !document.body.classList; } },
					{ src: 'plugin/markdown/marked.js', condition: function() { return !!document.querySelector( '[data-markdown]' ); } },
					{ src: 'plugin/markdown/markdown.js', condition: function() { return !!document.querySelector( '[data-markdown]' ); } },
					{ src: 'plugin/highlight/highlight.js', async: true, callback: function() { hljs.initHighlightingOnLoad(); } },
					{ src: 'plugin/search/search.js', async: true },
					{ src: 'plugin/zoom-js/zoom.js', async: true },
					{ src: 'plugin/notes/notes.js', async: true }
				]
			});
			
			// Slide number formatting can be configured using these variables:
			//  "h.v": 	horizontal . vertical slide number (default)
			//  "h/v": 	horizontal / vertical slide number
			//    "c": 	flattened slide number
			//  "c/t": 	flattened slide number / total slides
			Reveal.configure({ slideNumber: 'c' });
			
			// Control which views the slide number displays on using the "showSlideNumber" value:
			//     "all": show on all views (default)
			// "speaker": only show slide numbers on speaker notes view
			//   "print": only show slide numbers when printing to PDF
			//Reveal.configure({ showSlideNumber: 'speaker' });

		</script>

	</body>
</html>
